pkg/scheduler/scheduler_test.go

```diff
@@ -1009,7 +1009,10 @@ func Test_UnionedGVKs(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			registry := plugins.NewInTreeRegistry()
+			_, ctx := ktesting.NewTestContext(t)
+			ctx, cancel := context.WithCancel(ctx)
+			defer cancel()
```

Comment:
Why add a cancel? Are we spawning any go routine?

Reply:
Thanks for catch! I will remove the cancel.

Reply:
still not removed